content/markdown/04-intro-to-stats/04-08-linear-regression.Rmd

```diff
@@ -0,0 +1,468 @@
+---
+output:
+  pdf_document: default
+  html_document: default
+---
+# Linear Regression
+**Author:** Cole Brookson
+**Date:** 29 August 2022
+
+<em>What factors influence bison mass in a restored ecosystem?</em>
+
+One of the first chapters to a book on causal inference that I have recently read is entitled "All you need is regression". I thought this was an accurate way to start this section, because that statement is true! While we cannot cover all the theory of linear regression here, it is genuinely the case that nearly all classical frequentist statistical methods are simply a version of linear regression. A t-test? Actually just linear regression. ANOVA? Yup, also linear regression. Machine learning?? Many types of it are in fact just regression!
+
+So why then, you may ask, did we learn about t-tests before we learned about linear regression in stats class? Well, it turns out that as with almost all quantitative methods discussed in this section, linear regression is frightfully easy to implement, but actually not so simple "under the hood." 
+
+However, getting an intuitive understanding of the basics is very much possible and highly encouraged. Linear regression is probably the most fundamental of all topics you may learn in statistics, and it's worth thinking about it carefully. To be clear, you will likely re-visit this topic many times, likely over the course of years, adding and refreshing pieces of information that will eventually develop into a solid understanding. here, we attempt only to provide a brief introduction to the basics. 
+
+## What is Regression?
+
+A gross simplification of the process of linear regression would be that if we have a series of points between two continuous variables, we want to find a straight line that goes through as many of those points as possible. 
```

Comment:
'between two continuous variables' might be confusing. Maybe something like- if we want to predict the value of one continuous variable from the value of another